virtualization_view.m

```diff
@@ -465,6 +466,7 @@ - (NSSize)getVirtualMachineSizeInPixels
             }
         });
     }
+#endif
     return sizeInPixels;
```

Comment:
Is `sizeInPixels` initialized to 0 if `INCLUDE_TARGET_OSX_14` is undefined? Or is its content undefined? I don't know objc rules for local variable initialization.

Reply:
@cfergeau This value will become NSZeroSize. I handled this at
https://github.com/Code-Hex/vz/blob/46252d39b9cc1cfc2414239b0520437a06dbf245/virtualization_view.m#L432

Reply:
What I suggest is `__block NSSize sizeInPixels = NSZeroSize`, example/gui-linux segfaults for me without this explicit initialization.

Reply:
```
diff --git a/virtualization_view.h b/virtualization_view.h
index 09766a7..bdc83f5 100644
--- a/virtualization_view.h
+++ b/virtualization_view.h
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#import "virtualization_helper.h"
 #import <Availability.h>
 #import <Cocoa/Cocoa.h>
 #import <Virtualization/Virtualization.h>
```
is missing for `INCLUDE_TARGET_OSX_14`

Reply:
@cfergeau Thanks for your confirmation, I just applied it

Reply:
Oh I didn't see this comment, I will add this